tests/e2e/mu-plugins/e2e-rest-feature-flag.php

```diff
@@ -26,15 +26,15 @@ function () {
 			array(
 				'methods'             => WP_REST_Server::EDITABLE,
 				'callback'            => function ( WP_REST_Request $request ) {
-					$feature_flag_overrides = get_option( 'googlesitekit_e2e_feature_flags', array() );
+					$feature_flag_overrides = get_option( '_e2e_feature_flags', array() );
 
 					if ( $request['feature_value'] ) {
 						$feature_flag_overrides[ $request['feature_name'] ] = (bool) $request['feature_value'];
 					} else {
 						unset( $feature_flag_overrides[ $request['feature_name'] ] );
 					}
 
-					update_option( 'googlesitekit_e2e_feature_flags', $feature_flag_overrides );
+					update_option( '_e2e_feature_flags', $feature_flag_overrides );
```

Comment:
I'm assuming this change is to allow the flags to persist across resets (which may be needed), it was done this way intentionally (at least for now). If we need the flag to persist across resets, this will require additional changes to satisfy the ACs:

> * All features should be disabled by default in E2E until explicitly enabled
> * Enabled features should be reset after each test case (perhaps any time Site Kit is reset) so that no features are enabled again

In this case, we should probably add a `e2e/feature/reset` endpoint to set an empty array and hook that in `afterAll` in the bootstrap so that one test file doesn't influence another. Each test would still need to reset flags as necessary within its own file though, similar to how we handle the plugin reset.

I would leave it as-is for now though unless we can foresee needing it right away?

Reply:
Ah... okay, I thought it was done by accident. I'll revert it then.

Reply:
Updated.